### ISSUE TITLE
Can't skip accessibility access

### ISSUE THREAD

**Issue by RockwellB1** [d4rken/sdmaid-public#4206]
I wrote about this on the play store beta version as feedback, was told to come here.

Recently I can't use the app unless accessibility access is enabled. If it's not and I try to use the widget or quick access directly from the app, I get the setup pop-up telling me I need accessibility. I can select skip but then the app fails to work properly, can't even open the settings anymore, and I had to clear data and start fresh basically. With accessibility enabled everything seems to function fine but of course I have to wait or cancel the app wanting to clean everything up by using the accessibility feature.

This is happening on two different devices, both running Android 10. A Galaxy Note 20 Ultra and a Galaxy Tab S6. I'll check my old Nexus 6P too, since it's also on 10 but is rooted, and update with my findings. Checked the 6P and the app is working as it should. Not asking for accessibility and not forcing me to turn that seeing on.

If I can provide more info just let me know!

**Comment by d4rken** [comment]
Do you not have this menu? Is the option not working?

![2020-10-31 23 00 34](https://user-images.githubusercontent.com/1439229/97790896-0c16d280-1bcd-11eb-97a6-bf8d6fd91398.png)

**Comment by RockwellB1** [comment]
I do have that menu but if I skip turning on the accessibility then I run into problems. Can't open the settings for the app then and I have to clear app data and set things up again. When I try to open the settings I get a brief pop-up that I can't read and then the sidebar menu flickers back into view.

Which is something I just did and the app is no longer acting weird. This is the second or third time I've cleared the app data since this all started, so kind of weird it took several times to get it to stop acting up. This is on my phone anyways, I've not sorted out the tablet yet. I'll try clearing app data a few more times and report back.

**Comment by RockwellB1** [comment]
Ok, you've got me what was going on. The app on both devices is working properly now. So I suppose this issue can be marked as closed/deleted/whatever. Maybe something just really got messed up in the data and doing a couple app resets basically cleaned things up. First time I've had to clear data multiple times to fix an issue with an app, though maybe that's not uncommon.

Thanks for the help, love the app.
